portfolio/src/main/webapp/index.html

```diff
@@ -27,7 +31,7 @@ <h2><div id="changeText"></div></h2>
       <input type="number" id="comment-choice" min="1" max="10" value="1">
       <button id="commentBtn">Display</button>
       <ul id="comment-container"></ul>
-    </div>
-    <script src="script.js"></script>
+    </div>    
   </body>
+  <script src="script.js"></script>
```

Comment:
It felt very weird to me to put this outside of body: https://stackoverflow.com/questions/3037725/is-it-wrong-to-place-the-script-tag-after-the-body-tag

IMO, best practice is to put it in head but make it defer or async, but no need to do that here.